boards/arm/udoo_neo_full_m4/pinmux.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include "device_imx.h"
+
+static int udoo_neo_full_m4_init(struct device *dev)
```

Comment:
What about the GPIO pinmux config?

Reply:
Added LED GPIO pin config into pinmux.c + RDC access in soc.c